events/equeue/equeue.c

```diff
@@ -73,7 +73,7 @@ int equeue_create_inplace(equeue_t *q, size_t size, void *buffer) {
     q->queue = 0;
     q->tick = equeue_tick();
     q->generation = 0;
-    q->breaks = 0;
+    q->breaks = false;
```

Comment:
It's now "break", rather than "breaks", but that's a keyword. How about `break_requested`? Always good to name booleans as something that makes sense to be `true` or `false`.

Reply:
Ah! Good catch, that would explain why the CI was failing